components/icon/index.jsx

```diff
@@ -52,6 +52,7 @@ const Icon = ({
 				aria-hidden="true"
 				category={category}
 				className={classNames(className, 'slds-icon', {
+					'slds-icon--xx-small': size === 'xx-small',
```

Comment:
This is needed for the sort direction arrow and seems to be a valid option according to https://github.com/salesforce-ux/design-system/blob/master/ui/components/icons/base/_index.scss#L100